src/App.tsx

```diff
@@ -1,61 +1,135 @@
+import React, { useState } from 'react';
 import './App.scss';
+import { TodoList } from './components/TodoList';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { Todo } from './components/Types/Todo';
+import { User } from './components/Types/User';
+import 'bulma/css/bulma.css';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+function findUserById(userId: number): User | null {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
+
+function findUserByName(userName: string): User | null {
+  return usersFromServer.find(user => userName === user.name) || null;
+}
+
+const preparedTodos: Todo[] = todosFromServer.map(todo => {
+  return {
+    ...todo,
+    user: findUserById(todo.userId),
+  };
+});
 
 export const App = () => {
+  const [title, setTitle] = useState('');
+  const [selectedUser, setSelectedUser] = useState('');
```

Comment:
Better to save user id here. What u will di if user will have the same name

Reply:
fixed it